### DISCUSSION TITLE
running nvm use 12.22.8 giving me error "exit status 5: Access is denied." for non admin account

### DISCUSSION THREAD

**Discussion by kulkarniash-source** [coreybutler/nvm-windows#724] [Support] [answered]
Hi
I have 2 accounts on my windows machine, account admin, which is admin of this machine and account user which is user , 
I have installed nvm using account admin in c:\User\user\AppData\Roaming\nvm ,
I have also installed 2 node versions from admin account, 
When i go to DOS via admin account and run command nvm use 12.22.8 , and then do npm verion it works fine

But When i go to DOS via user account and run command nvm use 12.22.8 , i get "exit status 5: Access is denied." 

How can i make it work for user account without giving admin rights?

**Comment by coreybutler** [comment, marked as answer]
Please see the [common issues wiki](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#permissions-exit-1-exit-5-access-denied-exit-145).
